apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts

```diff
@@ -13,8 +13,8 @@ import {
 } from '../../../lib/db-types.js';
 import { type Lineitems, type Lti13CombinedInstance } from '../../lib/lti13.js';
 
-export const AssessmentRowSchema = AssessmentSchema.merge(
-  AssessmentSetSchema.pick({ abbreviation: true, name: true, color: true }),
+export const AssessmentRowSchema = AssessmentSchema.extend(
+  AssessmentSetSchema.pick({ abbreviation: true, name: true, color: true }).shape,
 ).extend({
   start_new_assessment_group: z.boolean(),
   assessment_group_heading: AssessmentSetSchema.shape.heading,
```

Comment:
Nit: [docs](https://zod.dev/api?id=extend) suggest using destructuring instead of extend or [in place of the deprecated merge](https://zod.dev/v4/changelog?id=deprecates-merge):
```suggestion
export const AssessmentRowSchema = z.object({
  ...AssessmentSchema.shape,
  ...AssessmentSetSchema.pick({ abbreviation: true, name: true, color: true }).shape,
  start_new_assessment_group: z.boolean(),
  assessment_group_heading: AssessmentSetSchema.shape.heading,
```

Reply:
Even better would be to fix this to stop using an extended type at all and instead use something like
```
AssessmentRowSchema = z.object({
  assessment: AssessmentSchema,
  assessment_set: AssessmentSetSchema,
  ...
})
```

Reply:
> Even better would be to fix this to stop using an extended type at all and instead use something like
> 
> ```
> AssessmentRowSchema = z.object({
>   assessment: AssessmentSchema,
>   assessment_set: AssessmentSetSchema,
>   ...
> })
> ```

That might be better done in a separate PR...